core/src/main/java/pl/agh/edu/ui/frame/bank/BankOfferFrame.java

```diff
@@ -0,0 +1,72 @@
+package pl.agh.edu.ui.frame.bank;
+
+import static pl.agh.edu.ui.utils.SkinColor.GRAY;
+import static pl.agh.edu.ui.utils.SkinFont.H4;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.ui.Button;
+import com.badlogic.gdx.scenes.scene2d.ui.ButtonGroup;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+
+import pl.agh.edu.GdxGame;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.data.loader.JSONBankDataLoader;
+import pl.agh.edu.engine.bank.BankAccount;
+import pl.agh.edu.ui.component.CustomScrollPane;
+import pl.agh.edu.ui.component.bankOffer.BankOffer;
+import pl.agh.edu.ui.component.label.LanguageLabel;
+import pl.agh.edu.ui.frame.BaseFrame;
+import pl.agh.edu.utils.LanguageString;
+
+public class BankOfferFrame extends BaseFrame {
+
+	public final List<BankOffer> buttonList = new ArrayList<>();
+	public final ButtonGroup<Button> buttonGroup = new ButtonGroup<>();
+	ScrollPane scrollPane;
+	BankAccount bankAccount = ((GdxGame) Gdx.app.getApplicationListener()).engine.bankAccountHandler.account;
+
+	public BankOfferFrame() {
+		super(new LanguageString("navbar.button.offer"));
+		String whiteFont = H4.getWhiteVariantName();
+		LanguageLabel changeBankLabel = new LanguageLabel(new LanguageString("bank.change"), whiteFont);
+		changeBankLabel.setBaseColor(GRAY);
+		mainTable.add(changeBankLabel).colspan(2).spaceBottom(BankOfferFrameStyle.subtitleSpace()).row();
+		Table contentRows = new Table();
+		scrollPane = new CustomScrollPane(contentRows, skin, "transparent");
+		JSONBankDataLoader.scenarios.forEach(scenario -> buttonList.add(new BankOffer(scenario)));
```

Comment:
ta cześć raczej powina być w zupełności na backendzie jeżeli chodzi o pobieranie jsonów

Reply:
to zostawić to na razie samo bez podpięcia czy jak to zrobić?